test/test.do

```diff
@@ -32,29 +32,29 @@ xtevent y eta, policyvar(z) panelvar(z) window(5)
 xtevent y eta, policyvar(z) timevar(z) window(5) 
 */
 
-* Testing noci, nosupt, nozeroline, nominus1label
+* Testing noci, nosupt, nozeroline, nonormlabel

```

Comment:
@Constantino-Carreto-Romero We need to document this change in the changelog in README.md please

Reply:
I documented it in the version's list of changes at https://github.com/JMSLab/xtevent/pull/168/commits/2fcc27a3f8de7fedb1ea255a269efd2b373631ad. I added that description to the draft of the release as well https://github.com/JMSLab/xtevent/releases.

Reply:
More detailed description about the option's changed name in https://github.com/JMSLab/xtevent/pull/168/commits/1514c451d0793982cdd071b7feb1381986faef45. I also added it to the release' draft.